.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,99 @@
+name: "🐞 Bug Report"
+description: "Tell us about something that's not working the way we (probably) intend."
+labels: ["Platform: React-Native", "Type: 🪲 Bug"]
+body:
+  - type: dropdown
+    id: environment
+    validations:
+      required: true
+    attributes:
+      label: "What React Native libraries do you use?"
+      description: "Select all options that describe your application."
+      multiple: true
+      options:
+        - "React Native without Frameworks"
+        - "React Navigation"
+        - "Hermes"
+        - "RN New Architecture"
+        - "Expo Application Services (EAS)"
+        - "Expo (mobile only)"
+        - "Expo Web"
+        - "Expo Router"
+        - "React Native Web"
+        - "React Native Navigation by Wix"
+        - "sentry.io SaS"
+        - "Self-Hosted Sentry"
```

Comment:
Q: Wdyt of moving the last two options on a separate dropdown?

Reply:
Yes, make sense, it doesn't fit in RN Libs section.